build/repo.props

```diff
@@ -8,4 +8,10 @@
   <PropertyGroup>
     <VSTestCLIRunSettings>RunConfiguration.TargetPlatform=x64</VSTestCLIRunSettings>
```

Comment:
I didn't realize anyone was still using this property. I dropped it when converting to parallel test assemblies. Does setting `<PlatformTarget>x64</PlatformTarget>` in the test csproj work?